src/main/java/org/example/homework14/AccordionInteractionTest.java

```diff
@@ -0,0 +1,29 @@
+package org.example.homework14;
+
+import org.openqa.selenium.*;
+import org.openqa.selenium.chrome.ChromeDriver;
+import java.util.concurrent.TimeUnit;
+
+public class AccordionInteractionTest {
+    public static void main(String[] args) {
+        WebDriver driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(10, TimeUnit.SECONDS);
+
+        try {
```

Comment:
Why did you put try-catch here? What kind of exception do you expect. Do not use try-catch for the logic, this construction is made for exceptions handling
Use it as a test and @AfterEach annotation to quit the driver